### PR TITLE
TRUNK-5519:Upgrade dom4j:dom4j

### PR DESCRIPTION
**Pull request by jwnasambu**
## Description of what I changed
I upgrade library dom4j:dom4j from 1.6.1 - 20040902.021138

## Issue I worked on
https://issues.openmrs.org/browse/TRUNK-5519

### CODE REVIEW

**Comment by jwnasambu** [comment]
@ruhanga and @mozzy11 locally it builds successfully but after making the PR travis fails how can I go about it?

File: pom.xml [hunk 1]
```diff
@@ -359,7 +359,7 @@
 			<dependency>
 				<groupId>dom4j</groupId>
 				<artifactId>dom4j</artifactId>
-				<version>1.6.1</version>
+				<version>20040902.021138</version>
```
Comment (HerbertYiga) [inline pom.xml]: do we really have this version under dom4j,kindly check well on the maven repository for dom4j!!

Comment (mozzy11) [reply pom.xml]: ` <version>1.6.1</version>` this seems to be the latest full release still

Comment (jwnasambu) [reply pom.xml]: @mozzy11  true. So what are we updating here? Kindly help me

Comment (mozzy11) [reply pom.xml]: you can close the PR , no update needed

**Comment by mozzy11** [comment]
thanks @samuelmale . i wonder where are you hibernating from these days

**Pull request closed without merging**